assets/tests/components/nav.test.tsx

```diff
@@ -71,7 +71,6 @@ describe("Nav", () => {
       </BrowserRouter>
     )
 
-    expect(result.queryByText("Mobile nav placeholder.")).not.toBeNull()
```

Comment:
Should probably also update the test description here and add an assertion that the mobile nav is rendered.